uber/decorators.py

```diff
@@ -24,6 +24,12 @@ def with_check(*args, **kwargs):
             return render('static_views/prereg_not_yet_open.html')
         elif c.AFTER_PREREG_TAKEDOWN and not c.AT_THE_CON:
             return render('static_views/prereg_closed.html')
+        elif c.OFFLINE_MODE:
+            with sa.Session() as session:
+                try:
+                    attendee = session.admin_account(cherrypy.session['account_id'])
```

Comment:
It would be better to replace this line with ``session.admin_attendee()`` since that does what you're doing behind the scenes.  Also there's no need to set an ``attendee`` variable if we're not using it.